packages/jaeger-ui/src/components/SearchTracePage/FileLoader.tsx

```diff
@@ -29,9 +31,7 @@ export default function FileLoader(props: FileLoaderProps) {
       beforeUpload={(file, fileList) => fileList.forEach(file => props.loadJsonTraces({ file }))}
       multiple
     >
-      <p className="ant-upload-drag-icon">
-        <FileAddOutlined />
-      </p>
+      <IoDocumentAttachOutline className="Dragger--icon" />
```

Comment:
Can't we do it like?

```html
<p className="ant-upload-drag-icon">
   <IoDocumentAttachOutline />
</p>
```

We don't need a new CSS class, I think.

Reply:
@yurishkuro are icon differences manageable?

New:
![image](https://github.com/jaegertracing/jaeger-ui/assets/94157520/b708ed2b-ba33-4f02-9f4e-1c2bc12c661b)


Old:
![image](https://github.com/jaegertracing/jaeger-ui/assets/94157520/5795ea54-b317-46df-962b-9b916a113346)

Reply:
Yeah, I think it's reasonable. But why did the color change?

Reply:
Maybe the color change should be fixed with this https://github.com/jaegertracing/jaeger-ui/pull/1771#discussion_r1327982089

Reply:
> Can't we do it like?
> 
> ```
> <p className="ant-upload-drag-icon">
>    <IoDocumentAttachOutline />
> </p>
> ```
> 
> We don't need a new CSS class, I think.

If I don't add css class the icon looks very small, and I can fix the color change with next patch.

Reply:
Can we reduce the size from 60px to 48px.
```
  font-size: 48px;
  color: #2DA6A2;
```
Got these properties from the `ant-upload-drag-icon` class. Maybe this can help here